src/service/world-api-adapter.js

```diff
@@ -25,11 +25,11 @@ module.exports = function (config) {
 
        token: store.get('epicenter.project.token') || '',
 
-       project: '',
+       project: undefined,
```

Comment:
So you can properly override them in the method options, otherwise $.extend() overrides the one you pass in the method with the empty string one